FlowCal/mef.py

```diff
@@ -308,38 +370,44 @@ def sc_fun(p,x):
             beads_model_str,
             beads_params_names)
 
-def plot_standard_curve(fl_channel,
+def plot_standard_curve(fl_au,
                         fl_mef,
                         beads_model,
                         std_crv,
+                        xscale='linear',
+                        yscale='linear',
                         xlim=None,
                         ylim=(1.,1e8)):
     """
     Plot a standard curve with fluorescence of calibration beads.
 
     Parameters
     ----------
-    fl_channel : array_like
-        Fluorescence of the calibration beads' subpopulations, in channel
-        numbers.
+    fl_au : array_like
+        Fluorescence of the calibration beads' subpopulations, in arbitrary
+        units.
     fl_mef : array_like
         Fluorescence of the calibration beads' subpopulations, in MEF
         units.
     beads_model : function
         Fluorescence model of the calibration beads.
     std_crv : function
-        The standard curve, mapping channel units to MEF units).
+        The standard curve, mapping arbitrary units to MEF units.
 
     Other parameters:
     -----------------
+    xscale : str, optional
+        Scale of the x axis, either ``linear`` or ``log``.
+    yscale : str, optional
+        Scale of the y axis, either ``linear`` or ``log``.
```

Comment:
I will not modify this since you agreed that we should keep this in log scale. But I wanted to answer some things.

> I'm also confused why there aren't negative values on the top plot. Do you only show negative numbers if necessary.

The negative interval in a logicle scale is "adaptive", in that it is changed depending on how negative the points are. This is not an axis limit effect, it affects scaling throughout all the range. For more details, check the paper.

This means that if there is no negative information, you will see that the axis starts at zero, but the rest of the plot will look pretty much like a log scale.

> I'm struggling to identify where the linear-to-log transition occurs on the scales, too.

It is a soft transition from log to linear, so most likely you won't be able to see it clearly.